app/src/main/AndroidManifest.xml

```diff
@@ -269,19 +269,6 @@
                     android:name="android.app.shortcuts"
                     android:resource="@xml/shortcuts"/>
         </activity>
-
-        <activity-alias
```

Comment:
This was the cause of the bug. `MainActivity_0` is identical to the default `MainActivity`